lib/AsyncStorage.js

```diff
@@ -100,6 +100,11 @@ const AsyncStorage = {
   ): Promise<null> {
     return new Promise((resolve, reject) => {
       RCTAsyncStorage.multiSet([[key, value]], function(errors) {
+        if (Platform.OS === 'ios' && typeof value !== 'string') {
+          console.warn(
```

Comment:
There are a lot of completely unrelated changes in this PR. Can we split them out to separate PRs? The scope of this one should only contain changes to this file.

Reply:
I had an issue to run the example app with Xcode 11, so I upgraded it. But good point, I'll create a different PR for that